Makefile

```diff
@@ -189,6 +189,7 @@ uninstall-dr-cluster: manifests kustomize ## Uninstall dr-cluster CRDs from the
 
 deploy-dr-cluster: manifests kustomize ## Deploy dr-cluster controller to the K8s cluster specified in ~/.kube/config.
 	cd config/dr-cluster/default && $(KUSTOMIZE) edit set image kube-rbac-proxy=$(RBAC_PROXY_IMG)
+	cd config/dr-cluster/default && $(KUSTOMIZE) edit set namespace $(OPERATOR_SUGGESTED_NAMESPACE)
```

Comment:
What is the intent of this change? IOW, what else is this needed for, or is there a larger change later here that is desired?

The OPERATOR_SUGGESTED_NAMESPACE is for setting the config map defaults. The make deploy-* is local instance deployment. If it is required to deploy to a non-default namespace the kustomize can be invoked directly, hence asking.